### PR TITLE
2-add-database-functions-that-are-needed-for-basic-gui

### PR DESCRIPTION
**Pull request by soerenlemke**

### CODE REVIEW

**Comment by soerenlemke** [comment]
@vyshnav-vinod made all the changes you mentioned.

File: database/getters.go [hunk 1]
```diff
@@ -33,3 +41,45 @@ func (g *Get) AllTableNames() ([]string, error) {
 
 	return tableNames, nil
 }
+
+func (g *Get) TableColumns(t string) ([]TableColumn, error) {
+	queryString := fmt.Sprintf("PRAGMA table_info(%s);", t)
```
Comment (vyshnav-vinod) [inline database/getters.go:46]: I am sorry. It appears that we dont need to use Prepare here, i went through the docs again and it seems Prepare is only used while inserting and other stuff. We can use the normal query here. We can merge this and i can make the change. Sorry for that

**Review by vyshnav-vinod — APPROVED** [review]

**Comment by vyshnav-vinod** [comment]
Looks good, lets merge this and i will change that Prepare statement. Sorry for that :)

File: database/getters.go (hunk 1 above)
Comment (soerenlemke) [reply database/getters.go:46]: No problem. I also researched a bit and it seems that this only makes sense when reusing the query for performance improvements, safety etc. Sounds good!

**Pull request merged**